src/main.cpp

```diff
@@ -2106,20 +2117,30 @@ void setup() {
     PinOledSDA = 21;
     PinOledSCL = 22;
 
-    PinBaroSDA = 13;
-    PinBaroSCL = 14;
+    // moving SCL SDA to different GPIO to avoid conflicts with mounted SD card on Lilygo T3 v2.1.1.6
+    PinBaroSDA = 3; //13;
+    PinBaroSCL = 4; //14;
```

Comment:
For me it works only the other way round. This seems to be a (quality) issue with the board, there are some reports on this in the web. Do you mind changing it here? Otherwise I might open a PR so that it can be changed in the web UI.

Reply:
sorry you mean it works with commented values? not sure I got it right. I'm getting a new board tomorrow, and can test other configurations. let me know thanks.

Reply:
For me it works to have
```suggestion
    PinBaroSDA = 4;
    PinBaroSCL = 3;
```
Of course with the wiring accordingly. It seems that due to quality issues some pins cannot switch within the specifications for I2C. This combination is more robust.